driver-core/src/main/com/mongodb/internal/connection/SaslAuthenticator.java

```diff
@@ -176,6 +176,7 @@ private void sendSaslContinueAsync(final BsonInt32 conversationId, final byte[]
 
     private BsonDocument createSaslStartCommandDocument(final byte[] outToken) {
         return new BsonDocument("saslStart", new BsonInt32(1)).append("mechanism", new BsonString(getMechanismName()))
+                .append("options", new BsonDocument("skipEmptyExchange", new BsonInt32(1)))
```

Comment:
The spec calls for the value of this option to be `true`, not `1`

Reply:
Done